lib/katello/tasks/clean_installed_packages.rake

```diff
@@ -0,0 +1,52 @@
+namespace :katello do
+  desc "Clean duplicate installed packages"
+  task :clean_installed_packages => :environment do
+    User.current = User.anonymous_admin
+
+    puts "Populating cache"
+    cache = PackageCache.new
+    cache.populate
+
+    total = ::Host.count
+    count = 1
+    ::Host.find_each do |host|
+      puts "Host #{count}/#{total}"
```

Comment:
minor thing: maybe print hostname in this line if we are printing one line per host anyway, just in case it gets stuck on a particular host so we can dig into it more easily